hack/cluster-openshift.sh

```diff
@@ -113,6 +114,10 @@ while [[ $# -gt 0 ]]; do
       OPENSHIFT_PERSISTENCE_DIR="$2"
       shift;shift
       ;;
+    -kn|--knative)
+      KNATIVE_ENABLED="true"
```

Comment:
Hardcoding "true" by mistake. You want:

```suggestion
      KNATIVE_ENABLED="$2"
```